packages/visual-editor/src/utils/i18n.ts

```diff
@@ -10,17 +10,9 @@ const fallbackT = (key: string) => key;
  *
  * @param key translation key
  */
-export const i18n = (key: string) => {
+export const i18n = (key: string): string => {
```

Comment:
How are we handling the complexities of i18n? We need to support plural, context, etc.